Requirement already satisfied: huggingface-hub in c:\users\andje\appdata\roaming\python\python311\site-packages (0.16.4)
Collecting huggingface-hub
  Using cached huggingface_hub-0.34.4-py3-none-any.whl.metadata (14 kB)
Requirement already satisfied: filelock in c:\users\andje\appdata\roaming\python\python311\site-packages (from huggingface-hub) (3.18.0)
Requirement already satisfied: fsspec>=2023.5.0 in c:\users\andje\appdata\roaming\python\python311\site-packages (from huggingface-hub) (2025.5.1)
Requirement already satisfied: packaging>=20.9 in c:\python311\lib\site-packages (from huggingface-hub) (23.2)
Requirement already satisfied: pyyaml>=5.1 in c:\python311\lib\site-packages (from huggingface-hub) (6.0.2)
Requirement already satisfied: requests in c:\python311\lib\site-packages (from huggingface-hub) (2.31.0)
Requirement already satisfied: tqdm>=4.42.1 in c:\users\andje\appdata\roaming\python\python311\site-packages (from huggingface-hub) (4.67.1)
Requirement already satisfied: typing-extensions>=3.7.4.3 in c:\python311\lib\site-packages (from huggingface-hub) (4.13.2)
Requirement already satisfied: colorama in c:\users\andje\appdata\roaming\python\python311\site-packages (from tqdm>=4.42.1->huggingface-hub) (0.4.6)
Requirement already satisfied: charset-normalizer<4,>=2 in c:\users\andje\appdata\roaming\python\python311\site-packages (from requests->huggingface-hub) (3.4.2)
Requirement already satisfied: idna<4,>=2.5 in c:\users\andje\appdata\roaming\python\python311\site-packages (from requests->huggingface-hub) (3.10)
Requirement already satisfied: urllib3<3,>=1.21.1 in c:\python311\lib\site-packages (from requests->huggingface-hub) (2.0.4)
Requirement already satisfied: certifi>=2017.4.17 in c:\users\andje\appdata\roaming\python\python311\site-packages (from requests->huggingface-hub) (2025.7.14)
Using cached huggingface_hub-0.34.4-py3-none-any.whl (561 kB)
Installing collected packages: huggingface-hub
  Attempting uninstall: huggingface-hub
    Found existing installation: huggingface-hub 0.16.4
    Uninstalling huggingface-hub-0.16.4:
      Successfully uninstalled huggingface-hub-0.16.4
  Rolling back uninstall of huggingface-hub
  Moving to c:\users\andje\appdata\roaming\python\python311\scripts\huggingface-cli.exe
   from C:\Users\andje\AppData\Local\Temp\pip-uninstall-lj1bm19w\huggingface-cli.exe
  Moving to c:\users\andje\appdata\roaming\python\python311\site-packages\huggingface_hub-0.16.4.dist-info\
   from C:\Users\andje\AppData\Roaming\Python\Python311\site-packages\~uggingface_hub-0.16.4.dist-info
  Moving to c:\users\andje\appdata\roaming\python\python311\site-packages\huggingface_hub\
   from C:\Users\andje\AppData\Roaming\Python\Python311\site-packages\~uggingface_hub
